modules/customizing-the-web-console-URL.adoc

```diff
@@ -53,6 +53,11 @@ spec:
 
 . Save the file to apply the changes.
 
+[NOTE]
```

Comment:
The wording of the note is fine, but I'm wondering if it might be better suited as the final step in this procedure or as a step in a "Next steps" section. For more information about the "Next steps" section, see the procedure template in the modular documentation guidelines: https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

Reply:
I think let it be a NOTE only to give the customer a heads up.